### ISSUE TITLE
Unable to "Add custom Helm Repo" with Chart Repo Crendentials but no TLS certificate

### ISSUE THREAD

**Issue by monnierant** [kontena/lens#7875]
**Describe the bug**
When we try to add a helm repo (we use a gitlab one) we are not able to click on the Add button if we doesn't set Key file, Ca file and certificate file even if they are not needed and even if we have check the Skip TLS certificate checks for the repository.

**To Reproduce**
1. Go to 'Settings > Kuerbnetes'
2. Click on 'Add Custom Helm Repo'
3. Enter a repo name like 'test-repo'
4. Enter a repository url protected by username and password
5. Check the 'Skip TLS certificate checks for the repository'
6. Let key file, Ca file and Certificate file empty
7. Enter 'Username' and 'Password' for login to the helm repo
8. The Add button stay greyed out so we can't add it

**Expected behavior**
We must get the Add button avaliable so we can click on it and add the helm repo

**Screenshots**
![2023-07-24_10h09_39](https://github.com/lensapp/lens/assets/3317226/1fdcfdb3-55e8-4dd7-8e59-9130648b9b6b)
![2023-07-24_10h09_50](https://github.com/lensapp/lens/assets/3317226/ff0c7087-b391-4d8c-8abd-0a9de2a6ab52)

**Environment (please complete the following information):**
- Lens Version: 2023.5.310801-latest
- OS: Windows 10
- Installation method (e.g. snap or AppImage in Linux): Windows installer

**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
```
No relevant log founds
```

**Kubeconfig:**
Quite often the problems are caused by malformed kubeconfig which the application tries to load. Please share your kubeconfig, remember to remove any secret and sensitive information.
```
Not related to kubeconfig
```

**Additional context**
I try with a dumb cert file and ca file but it try to check them and can't add the helm repo as cert files are empty

**Comment by pilopsoft-medvez** [comment]
yep, the bug still exists and still annoying. I hope this little workaround will help someone in the future.

As a workaround I saved the root CA used by my server. And I created a self-signed certificate with an easy `openssl` command. Then I removed the password from the keyfile. I think with the password it doesn't work either.

```
$ openssl s_client -showcerts -verify 5 -connect gitlab.example.com:443 < /dev/null
$ openssl req -x509 -sha256 -days 3650 -newkey rsa:2048 -keyout test.key -out test.crt
$ openssl rsa -in test.key -out test.unencrypted.key -passin pass:TYPE_YOUR_PASS
```
The `Key file` was the `test.unencrypted.key` the `Certificate fiile` is the `test.crt` and the `Ca file` is from gitlab server.
